composer/workflows/airflow_db_cleanup.py

```diff
@@ -53,11 +53,15 @@
 
 import airflow
 from airflow import settings
-from airflow.configuration import conf
-from airflow.jobs.base_job import BaseJob
+
+try:
+    from airflow.jobs import BaseJob
+except ImportError:
+    from airflow.jobs.base_job import BaseJob
+
 from airflow.models import DAG, DagModel, DagRun, Log, SlaMiss, \
     TaskInstance, Variable, XCom
-from airflow.operators.python import PythonOperator
+from airflow.operators.python_operator import PythonOperator
```

Comment:
There is no module python_operator in Airflow 2, am I right?

Reply:
Fixed in two separate files